python/ray/data/dataset.py

```diff
@@ -995,6 +995,11 @@ def filter(
             :meth:`Dataset.map_batches` might be faster. You can implement filter by
             dropping rows.
 
+        .. tip::
+            If you are using parquet and the filter is a simple predicate, you might
+            be able to speed it up by using filter pushdown, see
+            :ref:`Parquet row pruning <parquet_row_pruning>`.
```

Comment:
Nit: Our technical writing style guide suggests using contractions: https://developers.google.com/style/contractions
```suggestion
            If you're using parquet and the filter is a simple predicate, you might
            be able to speed it up by using filter pushdown, see
            :ref:`Parquet row pruning <parquet_row_pruning>`.
```

Reply:
Thanks :)